SshAgentLib/Util.cs

```diff
@@ -44,6 +46,27 @@ public static class Util
 
     static string assemblyTitle;
 
+    /// <summary>
+    /// Convert a SecureString to an asni string in the form of a PinnnedArray&lt;byte&gt;
+    /// </summary>
+    /// <param name="ss"></param>
+    /// <param name="func"></param>
+    /// <typeparam name="T"></typeparam>
+    /// <returns></returns>
+    public static PinnedArray<byte> ToAnsiArray(this SecureString ss)
+    {
+      if (ss == null) return null;
```

Comment:
No oneline ifs please.

```suggestion
      if (ss == null) {
        return null;
      }
```